pmm-app/src/pmm-qan/panel/QueryAnalytics.tools.ts

```diff
@@ -22,7 +22,7 @@ export function buildShareLink(from: number, to: number) {
 
   params.set('from', `${from}`);
   params.set('to', `${to}`);
-  params.set('orgId', config.bootData.user.orgId);
+  params.set('orgId', `${config.bootData.user.orgId}`);
```

Comment:
Is this really different from the one before? Or is it meant to type-convert from Integer to String?

Reply:
Yeah, we now get an error while trying to use a number there